pkg/steps/bundle_source.go

```diff
@@ -89,8 +89,8 @@ func (s *bundleSourceStep) bundleSourceDockerfile() (string, error) {
 	return strings.Join(dockerCommands, "\n"), nil
 }
 
-func (s *bundleSourceStep) getFullPullSpec(tag string) (string, error) {
-	is, err := s.imageClient.ImageStreams(s.jobSpec.Namespace()).Get(context.TODO(), api.StableImageStream, meta.GetOptions{})
+func getFullPullSpec(tag, namespace string, ic imageclientset.ImageStreamsGetter) (string, error) {
```

Comment:
Yuck. We have like 30 of these littered in the code. Can we please use `utils.ImageDigestFor`